### DISCUSSION TITLE
[Feature] view unnamed faces.

### DISCUSSION THREAD

**Discussion by chodthewacko** [immich-app/immich#10983] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

There should be a checkbox in 'explore -> people'  to only show people without names.

Naming people at the start mostly works great.
The explore -> people screen shows people ordered by number of assets, so you name people who you have the most pictures of (and are in theory the most important to you).

The problem is, people with names get shown "over" people without names.  Even if that name has only one face.
For a large library, you can easily reach 500 names, and then it becomes very tedious, if not impossible, to figure out which unnamed people have a large number of pictures.   Being able to not show named faces on 'explore->people' would fix that.

### Platform

- [x] Server
- [x] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#5801
